src/test/java/uk/gov/hmcts/reform/sendletter/tasks/reports/EmailSenderTest.java

```diff
@@ -0,0 +1,128 @@
+package uk.gov.hmcts.reform.sendletter.tasks.reports;
+
+import com.icegreen.greenmail.util.ServerSetupTest;
+import org.apache.commons.mail.util.MimeMessageParser;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.mail.MailException;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.util.ResourceUtils;
+import uk.gov.hmcts.reform.sendletter.jupiter.GreenMailExtension;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Properties;
+import javax.activation.DataSource;
+import javax.mail.Address;
+import javax.mail.internet.MimeMessage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.willThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class EmailSenderTest {
+
+    private static final String TEST_LOGIN = "test@localhost.com";
+    private static final String TEST_PASSWORD = "test_password";
+
+    private static final String SUBJECT = "email subject";
+    private static final String RECIPIENT_A = "recipient A <a@localhost>";
+    private static final String RECIPIENT_B = "b@localhost";
+    private static final File ATTACHMENT_FILE;
+    private static final Attachment ATTACHMENT_1;
+    private static final Attachment ATTACHMENT_2;
+
+    static {
```

Comment:
can we not put this in setUp method ?

Reply:
just wanted to load once per class, not once per test case. not bothered too much. should i move to setup method?

Reply:
We can use BeforeAll ?